catalog/models.py

```diff
@@ -38,4 +39,201 @@ def __str__(self) -> str:
         Returns:
             str: model data identifier.
         """
-        return self.name or self.identifier
\ No newline at end of file
+        return self.name or self.identifier
+
+
+class Catalogs (models.Model):
+    """Abstract model for Catalogs"""
+
+    image = models.CharField(
+        _('Color composition name'),
+        max_length=255,
+        unique=True
+    )
+
+    product = models.CharField(
+        _('Product scene'),
+        max_length=255,
+        null=True,
+        blank=True
+    )
+
+    satellite = models.ForeignKey(
+        'catalog.Satellite',
+        null=True,
+        blank=True,
+        on_delete=models.DO_NOTHING,
+    )
+
+    type = models.CharField(
+        _('Composition type'),
+        max_length=30,
+        null=True,
+        blank=True
+    )
+
+    date = models.DateField(
+        _('Scene Date')
+    )
+
+    cloud_cover = models.FloatField(
+        _('Percentage of cloud cover'),
+        null=True,
+    )
+
+    url_tms = models.CharField(
+        _('Tile path'),
+        max_length=511
+    )
+
+    path = models.CharField(
+        _('Image repository'),
+        max_length=511,
+    )
+
+    download_link = models.CharField(
+        _('Image repository'),
+        max_length=511,
+    )
+
+    download_available = models.BooleanField(
+        _("Download allowed"),
+        default=False,
+
+    )
+
+    max_native_zoom = models.IntegerField(
+        _('Maximum zoom scale'),
+        default=15,
+
+    )
+
+    co_cr = models.BigIntegerField(
+        _('Regional Coordenation code'),
+    )
+
+    ds_cr = models.CharField(
+        _('Regional Coordenation name'),
+        max_length=255,
+    )
+
+    co_funai = models.IntegerField(
+        _('Funai code'),
+    )
+
+    no_ti = models.CharField(
+        _('Name of Indigenous Lands'),
+        max_length=255,
+    )
+
+    municipio = models.CharField(
+        _('County'),
+        max_length=255,
+        blank=True,
+        null=True
+    )
+
+    sg_uf = models.CharField(
+        _('State aconymn'),
+        max_length=255,
+        blank=True,
+        null=True
+    )
+
+    resolution = models.IntegerField(
```

Comment:
Is it necessary?

Reply:
For this demand it is necessary.
Under study at CARD: https://xskylab.atlassian.net/browse/CMR-127